recipes/ptplot/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "ptplot" %}
+{% set version = "0.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ptplot-{{ version }}.tar.gz
+  sha256: 4d23b74bae59a217e37001fbcb3ef9a99b640e5c8dce504847f82697e46c149a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - bokeh >=2.3.3,<3
+    - nodejs >=14.17.0,<15
```

Comment:
Is nodejs really needed at run time? I'm no expert, just curious.

Reply:
Hey, @ocefpaf, good question! I have a custom bokeh extension as part of the package, and [until I figure out how to prebuild it](https://github.com/bokeh/bokeh/discussions/11603) users will need to have node installed in order to have bokeh runtime-compile it. Most of my users won't have a working node install or the technical savvy to get one, so I figured the easiest way to deal with that was to make it a dependency.